.mailmap

```diff
@@ -500,10 +502,10 @@ Tyler Larson <talltyler@gmail.com>
 Ujjwal Sharma <ryzokuken@disroot.org> <ryzokuken@igalia.com>
 Ujjwal Sharma <ryzokuken@disroot.org> <usharma1998@gmail.com>
 Uttam Pawar <upawar@gmail.com> <uttam.c.pawar@intel.com>
+Viero Fernando <vierofernando9@gmail.com>
```

Comment:
Whoops, this needs to be changed:

```suggestion
Viero Fernando <vierofernando9@gmail.com> <60427892+vierofernando@users.noreply.github.com>
```

Otherwise it will still show `null` because it doesn't know to map `<60427892+vierofernando@users.noreply.github.com>` to `Viero Fernando <vierofernando9@gmail.com>`.

Reply:
done.